lib/tessel/deploy.js

```diff
@@ -334,13 +334,14 @@ actions.tarBundle = function(opts) {
 
           return rules.concat(patterns);
         }, []).map(function(rule) {
-
+          // Ensure it uses proper paths for OS
+          rule = path.normalize(rule);
           if (rule[rule.length - 1] === '/') {
-            rule += '**/*.*';
+            rule += path.normalize('**/*.*');
           }
 
           if (rule[rule.length - 1] !== '*' && rule.indexOf('.') === -1) {
-            rule += '/**/*.*';
+            rule += path.normalize('/**/*.*');
           }
 
```

Comment:
Interesting—I might be confused about which tests existed before and after appveyor was added, in relation to when that code was landed. The fix is fine to get green, even if they've been removed in #435